ocaml/xapi/extauth_plugin_ADwinbind.ml

```diff
@@ -226,41 +226,54 @@ module Ldap = struct
       try Ok (of_string str)
       with _ -> Error (ldap "invalid value for key '%s'" k)
     in
+    let get_int_with_default of_string ~k ~default =
+      match get_int of_string k with Ok x -> Ok x | Error _ -> Ok default
```

Comment:
The name `get_int` could be misleading because the return type is determined by the first argument. `get_int` is indeed polymorphic and could be called `get` and `get_with_default` instead.